src/discussions/comments/CommentsView.jsx

```diff
@@ -22,8 +22,13 @@ function CommentsView({ intl }) {
   const dispatch = useDispatch();
   const thread = useSelector(selectThread(postId));
   const comments = useSelector(selectThreadComments(postId));
+  const hasMorePages = useSelector(store => store.comments.pagination[postId]?.hasMorePages || false);
+  const currentPage = useSelector(store => store.comments.pagination[postId]?.currentPage || null);
+  const onLoadMoreCommentsClick = () => dispatch(fetchThreadComments(postId, { page: currentPage + 1 }));
```

Comment:
nit: rename to handleLoadMoreComments

The source of the event could be a click a touch or a keyboard action, and the "onEvent" patterns is used for parameters to which you can provide a "handlerEvent" function to keep things easier to follow.